generators/app/index.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+var yeoman       = require('yeoman-generator');
+var slugify      = require('underscore.string').slugify;
+var normalizeUrl = require('normalize-url');
+
+function filterProjectName (name) {
+    return slugify(name.replace(/^testcafe(-|\s)browser(-|\s)provider(-|\s)/i, ''));
+}
+
+module.exports = yeoman.generators.Base.extend({
+    prompting: function () {
+        var done = this.async();
+        var gen  = this;
+
+        var prompts = [
+            {
+                name:    'providerName',
+                message: 'How do you want to name your browser provider?',
+                default: filterProjectName(this.appname) || void 0,
+                filter:  filterProjectName
+            },
+            {
+                name:     'githubUsername',
+                message:  'What is your GitHub username?',
+                store:    true,
+                validate: function (name) {
+                    return name.length ? true : 'You should provide a username';
+                }
+            },
+            {
+                name:    'homepage',
+                message: 'What is the URL of your browser provider homepage?',
```

Comment:
amm... I would probably get stuck on this question.
So, I'm about to create a browser provider... And it already needs to have a homepage?